2022-tavasz/kotlin-springboot-workshop/README.md

```diff
@@ -0,0 +1,14 @@
+# Spring boot - Kotlin workshop 2022
+
+## Hogyan futtassuk
+
+* Nyissuk meg megfelelő feljesztőeszközzel, majd futtassuk, mint Spring alkalmazás.
+* Vagy: A `rocketmanager` mappában adjuk ki a `mvnw clean spring-boot:run` parancsot.
+* Vagy: Amennyiben van a gépen telepítve `docker`, adjuk ki a `docker-compose up --build` parancsot.
+
+## További anyagok
+
+* Videófelvétel: [link](https://www.youtube.com/watch?v=DPgjYITMjkA)
```

Comment:
Maybe we might have to transfer this video under the Kir-Dev account so that all these stuff are available at one place

https://www.youtube.com/channel/UCkpMTj9qST_7RDt2YL4RUEw

I can grant you permission for uploading if you give me your email address or something

Reply:
Done